pttl.lua

```diff
@@ -0,0 +1,21 @@
+-- pttl.lua: => Arguments: [value]
+-- pttl.lua returns provided keys's ttls if all their values match the input. 
+
+-- Check all keys values matches provided input.
+local values = redis.call("mget", unpack(KEYS))
+for i, _ in ipairs(KEYS) do
+	if values[i] ~= false and values[i] ~= ARGV[1] then
+		return redis.error_reply("redislock: lock not held")
+	end
+end
+
+-- Find and return shortest TTL among keys.
+local minTTL = 0
```

Comment:
this should start as `-1`

Reply:
Please note that pttl will return negative value in case of error (like if the key does not exists). Therefore minTTL might never be set. Setting it to zero avoid checking for potential -1 before returning it.
Is there any particular reason to set it to -1 (and therefore add another if statement) ? 
I am missing something ?